README.md

```diff
@@ -20,19 +20,19 @@ results before checking in the modified code.
 
 ## Installation
 
-OS X users with homebrew should be able to install via ```brew install dfix``` for the latest stable release or ```brew install dfix --HEAD``` for the latest git master branch.
+OS X users with homebrew should be able to install via `brew install dfix` for the latest stable release or `brew install dfix --HEAD` for the latest git master branch.
 
-Other users should manually install, e.g. on *nix systems:
+Other users should manually install, e.g. on \*nix systems:
 
-* ```git clone https://github.com/Hackerpilot/dfix && git submodule update --init```
-* ```cd dfix```
-* ```git checkout v0.3.1``` if you want the stable release
-* ```make``` to build
-* ```make test``` to test
-* either add the ```bin``` directory to your path or copy to another directory that is on your path.
+* `git clone https://github.com/dlang-community/dfix && git submodule update --init`
+* `cd dfix`
+* `git checkout v0.3.2` if you want the stable release
```

Comment:
latest tag is v0.3.5

Reply:
You can get the latest* tag via the GitHub API:
```
curl -sL 'https://api.github.com/repos/dlang-community/dfix/tags' | jq -r '.[0].name'
```

*though there's no guarantee that this would be the latest tag according to SemVer.

Reply:
Or using git https://stackoverflow.com/a/17414170. If it was a one liner this is what should be indicated. Considering how often is updated  dfix i think that we can indicate a version and eventually it will be updated in a further PR.